examples/megacli.cpp

```diff
@@ -2074,6 +2074,253 @@ static void store_line(char* l)
     line = l;
 }
 
+#if __cplusplus >= 201100L
+
+class FileFindCommand : public Command
```

Comment:
I'm aware this is not the only declaration added directly to the .cpp, but I'd prefer to keep declaration in headers and definitions in sources. Not a biggie :)